infoschema/perfschema/tables_test.go

```diff
@@ -57,8 +57,6 @@ func TestPerfSchemaTables(t *testing.T) {
 }
 
 func TestTiKVProfileCPU(t *testing.T) {
-	t.Parallel()
```

Comment:
The common way to fix it is to create a new test file called `xxx_serial_test.go` in same package.

Reply:
Done